specification.markdown

```diff
@@ -448,116 +448,191 @@ This table clarifies if a Mandatory EBML Element MUST be written, according to i
-Default      | 1
-Type         | Unsigned Integer
-Description  | The version of DocType interpreter used to create the EBML Document.
-
-Name         | DocTypeReadVersion
-:------------|:------------------
-Level        | 1
-EBML ID      | `0x4285`
-Mandatory    | Yes
-Multiple     | No
-Range        | -
-Default      | 1
-Type         | Unsigned Integer
-Description  | The minimum DocType version an interpreter has to support to read this EBML Document. The DocTypeReadVersion MUST be less than or equal to DocTypeVersion.
+### EBML Element
+
+name: `EBML`
+
+level: 0
+
+id: `0x1A45DFA3`
+
+minOccurs: 1
+
+maxOccurs: 1
+
+type: `Master Element`
+
+description: Set the `EBML` characteristics of the data to follow. Each `EBML Document` has to start with this.
+
+### EBMLVersion Element
+
+name: `EBMLVersion`
+
+level: 1
+
+id `0x4286`
+
+minOccurs: 1
+
+maxOccurs: 1
+
+range:  1
+
+default: 1
+
+type: Unsigned Integer
+
+description: The version of `EBML Writer` used to create the `EBML Document`.
+
+### EBMLReadVersion Element
+
+name: `EBMLReadVersion`
+
+level: 1
+
+id: `0x42F7`
+
+minOccurs: 1
+
+maxOccurs: 1
+
+range:  1
+
+default: 1
+
+type: Unsigned Integer
+
+description: The minimum `EBML` version an `EBML Reader` has to support to read this `EBML Document`. The `EBMLReadVersion Element` MUST be less than or equal to `EBMLVersion`.
+
+### EBMLMaxIDLength Element
+
+name: `EBMLMaxIDLength`
+
+level:  1
+
+id `0x42F2`
+
+minOccurs: 1
+
+maxOccurs: 1
+
+range:  >3
+
+default: 4
+
+type: Unsigned Integer
+
+description: The `EBMLMaxIDLength Element` stores the maximum length in octets of the `Element IDs` to be found within the `EBML Bod`y. An `EBMLMaxIDLength Element` value of four is RECOMMENDED, though larger values are allowed.
+
+### EBMLMaxSizeLength Element
+
+name: `EBMLMaxSizeLength`
+
+level: 1
+
+id `0x42F3`
+
+minOccurs: 1
+
+maxOccurs: 1
+
+range:  >0
+
+default: 8
+
+type: Unsigned Integer
+
+description: The `EBMLMaxSizeLength Element` stores the maximum length in octets of the expression of all `Element Data Sizes` to be found within the `EBML Body`. To be clear the `EBMLMaxSizeLength Element` documents the maximum 'length' of all `Element Data Size` expressions within the `EBML Body` and not the maximum 'value' of all `Element Data Size` expressions within the `EBML Body`. `EBML Elements` that have an `Element Data Size` expression which is larger in octets than what is expressed by `EBMLMaxSizeLength ELEMENT` SHALL be considered invalid.
+
+### DocType Element
+
+name: `DocType`
+
+level: 1
+
+id `0x4282`
+
+minOccurs: 1
+
+maxOccurs: 1
+
+type: String
+
+description: A string that describes and identifies the content of the `EBML Body` that follows this `EBML Header`.
+
+### DocTypeVersion Element
+
+name: `DocTypeVersion`
+
+level: 1
+
+id `0x4287`
+
+minOccurs: 1
+
+maxOccurs: 1
+
+range:  -
+
+default: 1
+
+type: Unsigned Integer
+
+description: The version of DocType interpreter used to create the EBML Document.
```

Comment:
updated and squashed into https://github.com/Matroska-Org/ebml-specification/pull/101/commits/229e83b3d18eb5dc6162343fa2f65849ee7ae4d9